### ISSUE TITLE
branch.subscribe() calls the listener when the app resumes

### ISSUE THREAD

**Issue by LonelyCpp** [BranchMetrics/react-native-branch-deep-linking-attribution#617]
Our app has a custom deeplink scheme (eg: `myapp://article`)

Manifest looks like this - 
```
<intent-filter>
  <data android:host="article" android:scheme="myapp" />
  
  <action android:name="android.intent.action.VIEW" />
  <category android:name="android.intent.category.DEFAULT" />
  <category android:name="android.intent.category.BROWSABLE" />
</intent-filter>
```

Branch gives a callback when this deeplink is triggered. 

But it also triggers again when the app goes to the background and switches back to the foreground.

Is there anything I missed?

**Comment by Solid-Metal** [comment]
yep, same problem with me as well with 5.0.1, is there any fix for this ?

**Comment by Svarto** [comment]
same issue here, this has been open for more than a year - anyone run into the same or know of a solution?

Seems to only happen on Android for me.

**Comment by hassanrazavd** [comment]
happening on android as well as ios for me . anyone found any solution?

**Comment by alekhinen** [comment]
Having the exact same issue on Android. The same deeplink is getting opened every single time the android app is backgrounded / foregrounded. Not sure how to solve this. Configuration / setup on my app is more-or-less identical with the official docs.
